docs/Usage.md

```diff
@@ -101,7 +101,7 @@ the `EXP_CLUSTER_RESOURCE_SET` is required if you want to deploy CNI using clust
 
 Once you have access to a management cluster, you can initialize Cluster API with the following:
 ```
-clusterctl init --infrastructure proxmox --ipam in-cluster
+clusterctl init --infrastructure proxmox --ipam in-cluster --core cluster-api:v1.5.3
```

Comment:
@avorima 
Do you agree with pinning the capi version in the quick-start docs?
while our compatibility versions matrix mentions that v1.6 not yet supported?
https://github.com/ionos-cloud/cluster-api-provider-proxmox/tree/main?tab=readme-ov-file#compatibility-with-cluster-api-and-kubernetes-versions


In my point of view we should plan the support for v1.6 very soon.

Reply:
Let's wait for @avorima feedback.

Reply:
Yes, since those commands are meant to be copied into a terminal.

Reply:
ok then